### PR TITLE
TokenRequest feature documentation

### PR DESCRIPTION
**Pull request by mikedanese**
/assign @awly @cjcullen

### CODE REVIEW

File: content/en/docs/tasks/configure-pod-container/configure-service-account.md [hunk 1]
```diff
@@ -250,12 +250,54 @@ spec:
...
-## Service Account Volume Projection
+## Service Account Token Volume Projection
 
-Kubernetes 1.11 and higher supports a new way to project a service account token into a Pod.
-You can specify a token request with audiences, expirationSeconds. The service account token
-becomes invalid when the Pod is deleted. A Projected Volume named
-[ServiceAccountToken](/docs/concepts/storage/volumes/#projected) requests and stores the token.
+{{< note >}}
+**Note:** This ServiceAccountTokenVolumeProjection is __beta__ in 1.12 and
+enabled by passing the following flags to the API server:
```
Comment (awly) [inline content/en/docs/tasks/configure-pod-container/configure-service-account.md]: `passing all of the following flags`

File: content/en/docs/tasks/configure-pod-container/configure-service-account.md [hunk 2]
```diff
@@ -250,12 +250,54 @@ spec:
...
+  - name: vault-token
+    projected:
+      sources:
+      - serviceAccountToken:
+          path: vault-token
+          expirationSeconds: 7200
+          audience: vault
+```
+
+The kubelet will requests and stores the token on behalf of the pod, make the
```
Comment (awly) [inline content/en/docs/tasks/configure-pod-container/configure-service-account.md]: "will request and store"

File: content/en/docs/tasks/configure-pod-container/configure-service-account.md [hunk 3]
```diff
@@ -250,12 +250,54 @@ spec:
...
+token avaialble to the pod at a configurable file path, and refresh the token as
+it approaches expiration.
```
Comment (awly) [inline content/en/docs/tasks/configure-pod-container/configure-service-account.md]: mention the refresh threshold
also mention that pod is responsible for re-loading the updated token

File: content/en/docs/tasks/configure-pod-container/configure-service-account.md [hunk 4]
```diff
@@ -250,12 +250,54 @@ spec:
...
-[ServiceAccountToken](/docs/concepts/storage/volumes/#projected) requests and stores the token.
+{{< note >}}
+**Note:** This ServiceAccountTokenVolumeProjection is __beta__ in 1.12 and
+enabled by passing the following flags to the API server:
+
+```
+* --service-account-issuer
+* --service-account-signing-key-file
+* --service-account-api-audiences
+```
```
Comment (tfogo) [inline content/en/docs/tasks/configure-pod-container/configure-service-account.md]: This code block is formatted a bit weirdly. Looks like you want bullets with the flags code formatted? In which case try this instead:

```
* `--service-account-issuer`
* `--service-account-signing-key-file`
* `--service-account-api-audiences`
```

File: content/en/docs/tasks/configure-pod-container/configure-service-account.md [hunk 5]
```diff
@@ -250,12 +250,54 @@ spec:
...
 
-## Service Account Volume Projection
+## Service Account Token Volume Projection
 
-Kubernetes 1.11 and higher supports a new way to project a service account token into a Pod.
-You can specify a token request with audiences, expirationSeconds. The service account token
-becomes invalid when the Pod is deleted. A Projected Volume named
-[ServiceAccountToken](/docs/concepts/storage/volumes/#projected) requests and stores the token.
+{{< note >}}
+**Note:** This ServiceAccountTokenVolumeProjection is __beta__ in 1.12 and
```
Comment (tfogo) [inline content/en/docs/tasks/configure-pod-container/configure-service-account.md]: You could also add the version shortcode to the top of this section:

`{{< feature-state for_k8s_version="v1.12" state="beta" >}}`

**Comment by mikedanese** [comment]
Updated PTAL.

**Comment by tfogo** [comment]
Thanks @mikedanese 
/approve

**Pull request merged**